app/controllers/guitar_gallery_controller.rb

```diff
@@ -0,0 +1,25 @@
+class GuitarGalleryController < ApplicationController
+    def index
+      render :json => [
+          {
+              'name':'guitar1',
+              'url':'https://rukminim1.flixcart.com/image/416/416/acoustic-guitar/x/8/w/topaz-blue-signature-original-imaefec7uhypjdr9.jpeg?q=70',
+              'price':'100',
+              'description':'blablabla 1'
+          },
+          {
+              'name':'guitar2',
+              'url':'https://shop.brianmayguitars.co.uk/user/special/content/Antique%20Cherry%20a.jpg',
+              'price':'200',
+              'description':'blablabla 2'
+          },
+          {
+              'name':'guitar3',
+              'url':'https://cdn.mos.cms.futurecdn.net/Yh6r74b8CAj2jbdf2FAhq4-970-80.jpg.webp',
+              'price':'300',
+              'description':'blablabla 3'
+          }
+      ]
+    end
+  end
+  
```

Comment:
Can you also setup Rubocop? this is the RoR linter we are using in Foreman,
https://docs.rubocop.org/rubocop/1.37/index.html https://sourcelevel.io/blog/rubocop-how-to-install-and-configure

Reply:
This is addressed in another PR: #15 
I Thought we could merge it first, and then add the configuration here :)
What do you think? @Ron-Lavi

Reply:
Rubocop PR merged, and rebased this PR on the Rubocop setup so it would be used in this PR as well 
I think this conversation can be resolved 👍🏼